### ISSUE TITLE
Frequency Table storing results?

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#4081]
The problem first, then the route and why I did this.
I did a 2-way frequency table of counts (not html).  This was of project by Country (for 5 countries).  Quite interesting.
Then I ran it again and opted to Store Results.  It only half-worked in that the results seemed to be only for the first factor - project.  I expected them to be for both factors.  I then realised that I did have all the tables - 4 of them - which is more than I needed.  And the visible one is a margin and not the 2-way table.

I suggest usually it is nice to see the margins in the output, but not necessarily to then store them all.

So 2 changes are suggested:
a) When margins are stored (i.e. 4 data frames are produced for a 2-way table), then could we have the tables in the **opposite order**, so the current one is then the 2-way table.
b) When you opt to **Store Results**, if margins are not ticked all is sweetness and light.  If margins are ticked then a second checkbox is enabled that says Store Margins.  By default it is un-ticked.

**Comment by dannyparsons** [comment]
yes this seems sensible. Instead of reordering the sheets we could show the first new one by default instead of the last.

**Comment by lilyclements** [comment]
@Ag-Derek can you assign someone to this? It would be useful to just check for now if these are still outstanding issues.

**Comment by Ag-Derek** [comment]
@Emmanuel-Afrifa can have a look at this
Thanks

**Comment by Emmanuel-Afrifa** [comment]
@rdstern I was investigating this issue and I found that the tables were produced and everything works well `(Two Way Frequencies dialog)`. The only potential issue was that for the table option, the results were being stored as `summary` hence when you go to the `Use Table` dialog, you do not have access to the table that was created. But the main 2 way frequencies table was created.

I wanted to double-check with you if your issue had been resolved (or if I have misunderstood the original issue) and whether it'll be necessary for me to change the save type for the tables from `summary` to `table`.
